pyproject.toml

```
@@ -106,10 +106,10 @@ test-docker = [
 ]
 
 [project.urls]
-Homepage = "https://my.project/"
-Changelog = "https://my.project/CHANGELOG"
-Documentation = "https://my.project/docs/"
-Issues = "https://my.project/issues"
+Homepage = "https://github.com/oracle/macaron"
+Changelog = "https://github.com/oracle/macaron/blob/main/CHANGELOG.md"
+Documentation = "https://oracle.github.io/macaron/"
```

```suggestion
Documentation = "https://oracle.github.io/macaron"
```
To be consistent with other URLs in this section.

---

`macaron` is a path parameter. That's why a `/` is automatically added. I can add `index.html` to the path.

---

Oh okay, if it's intended then I am all good with it. Thanks

---

See commit [d0deb1d](https://github.com/oracle/macaron/pull/867/commits/d0deb1dfa696813029f64d1abef0136f5a862046).